blocks/init/components/scroll-to-top/scroll-to-top.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Template for the Scroll To Top Component.
+ *
+ * @since 1.0.0
+ * @package Eightshift_Boilerplate\Blocks.
+ */
+
+namespace Eightshift_Boilerplate\Blocks;
+
+?>
+
+<a href='#' class='js-scroll-to-top'>
+  <?php echo esc_html_e( 'To Top', 'eightshift-boilerplate' ); ?>
```

Comment:
```suggestion
  <?php esc_html_e( 'To Top', 'eightshift-boilerplate' ); ?>
```

I thought that scroll to top is usually an icon, not text

Reply:
yea it is but it is up to project implementation